README.md

```diff
@@ -12,11 +12,42 @@ $ npm install eslint-loader
 
 In your webpack configuration
 
-```js
+```javascript
+module.exports = {
+  // ...
+  module: {
+    loaders: [
+      {test: /\.js$/, loader: "eslint-loader", exclude: /node_modules/}
+    ]
+  }
+  // ...
+}
+```
+
+When using with transpiling loaders (like `babel-loader`), make sure they are in correct order
```

Comment:
Maybe it is even better to leave only `preLoaders` section and do not mention loaders ordering, to keep docs less verbose